sdk/tests/js/webgl-test-utils.js

```diff
@@ -2913,6 +2913,23 @@ var requestAnimFrame = function(callback) {
   _requestAnimFrame.call(window, callback);
 };
 
+var _requestVidFrameCallback = null;
+
+
+/**
+ * Provides video.requestVideoFrameCallback in a cross browser way.
+ * Returns a property, or undefined if unsuported.
+ */
+var getRequestVidFrameCallback = function() {
+  if (_requestVidFrameCallback === null) {
+    var vid = document.createElement('video');
+    _requestVidFrameCallback =
+      getPrefixedProperty(vid, "requestVideoFrameCallback");
```

Comment:
Is it necessary to try all of the browser-specific prefixes? Since the current web ecosystem development process discourages shipping prefixed APIs, it would be better to just look for the specified name.

In this case, can this be simpler?

`  return HTMLVideoElement.prototype["requestVideoFrameCallback"]
`
? Works in both Chrome Canary (has support) and Stable (doesn't have support).

Reply:
It isn't necessary, and your proposed method is much simpler :) My experience with JS is quite limited, and I didn't know this was an option.

I've also added the support to the 1.0.1 and 1.0.2 versions, now that I'm not reliant on the "getPrefixedProperty".

I'm also happy to directly inline `HTMLVideoElement.prototype["requestVideoFrameCallback"]`, since it's only used only once per file, but maybe there will be other users in the future and the "get" function makes sense.